fixed-hash/src/lib.rs

```diff
@@ -10,7 +10,6 @@
 
 // Re-export liballoc using an alias so that the macros can work without
 // requiring `extern crate alloc` downstream.
-#[cfg(not(feature = "std"))]
```

Comment:
could also be `#[cfg(any(no_std, feature = "rustc-hex"))]` but `std` is using `rustc-hex` so I re-exported `alloc` for all instead,